.github/workflows/scripts/check-license-header.sh

```diff
@@ -40,12 +40,12 @@ fi
 
 paths_with_missing_license=( )
 
-file_excludes=".license_header_template
-.licenseignore"
-if [ -f .licenseignore ]; then 
-  file_excludes=$file_excludes$(printf '\n')$(cat .licenseignore)
+if [[ -f .licenseignore ]]; then
+  file_paths=$(tr '\n' '\0' < .licenseignore | xargs -0 -I% printf '":(exclude)%" '| xargs git ls-files ":(exclude).licenseignore" ":(exclude).license_header_template" )
+else
+  file_paths=$(git ls-files ":(exclude).license_header_template" )
 fi
```

Comment:
The second `xargs` confused me a little bit until I realized that it only causes `git ls-files` to be called a single time.  Maybe worth pulling out the computation of the exclude list to make it more readable? Something along the lines of the following.
 
```suggestion
if [[ -f .licenseignore ]]; then
  exclude_list="':(exclude).licenseignore' ':(exclude).license_header_template' $($(tr '\n' '\0' < .licenseignore | xargs -0 -I% printf '":(exclude)%")"
else
  exclude_list=":(exclude).license_header_template"
fi
file_paths=$(git ls-files $exclude_list)
```

Reply:
I agree though I would prefer a follow up PR to unblock the CI on other repos